mlir/test/fusion/e2e/lit.site.cfg.py.in

```diff
@@ -0,0 +1,75 @@
+@LIT_SITE_CFG_IN_HEADER@
+
+import sys
+import subprocess
+
+config.llvm_obj_root = "@LLVM_BINARY_DIR@"
+config.llvm_tools_dir = "@LLVM_TOOLS_DIR@"
+config.llvm_shlib_dir = "@SHLIBDIR@"
+config.llvm_shlib_ext = "@SHLIBEXT@"
+config.lit_tools_dir = "@LLVM_LIT_TOOLS_DIR@"
+config.python_executable = "@Python3_EXECUTABLE@"
+config.mlir_src_root = "@MLIR_SOURCE_DIR@"
+config.mlir_obj_root = "@MLIR_BINARY_DIR@"
+config.mlir_runner_utils_dir = "@MLIR_RUNNER_UTILS_DIR@"
+config.mlir_tools_dir = "@MLIR_TOOLS_DIR@"
+config.mlir_rock_tools_dir = "@ROCMLIR_TOOLS_DIR@"
+config.linalg_test_lib_dir = "@MLIR_DIALECT_LINALG_INTEGRATION_TEST_LIB_DIR@"
+config.mlir_lib_dir = "@MLIR_LIB_DIR@"
+config.conv_validation_wrapper_library_dir = "@MLIR_CONV_VALIDATION_WRAPPER_LIBRARY_DIR@"
+config.allow_e2e_tests = @ROCK_E2E_TEST_ENABLED@ or @ROCMLIR_DRIVER_PR_E2E_TEST_ENABLED@
+config.enable_rocm_runner = @MLIR_ENABLE_ROCM_RUNNER@
+if config.enable_rocm_runner:
+    config.available_features.add('rocm-runner')
+config.enable_rock_driver_e2e_test = @ROCMLIR_DRIVER_E2E_TEST_ENABLED@
+config.enable_rock_driver_pr_e2e_test = @ROCMLIR_DRIVER_PR_E2E_TEST_ENABLED@
+config.enable_rock_driver_any_e2e_test = config.enable_rock_driver_pr_e2e_test or config.enable_rock_driver_e2e_test
+if config.enable_rock_driver_any_e2e_test:
+    config.available_features.add('rock-any-e2e')
+config.enable_bindings_python = @MLIR_BINDINGS_PYTHON_ENABLED@
+config.random_data = "@MLIR_RANDOM_DATA@"
+config.rocmlir_gen_flags = "@ROCMLIR_GEN_FLAGS@"
+config.populate_validation = "@MLIR_POPULATE_VALIDATION@"
+config.rocm_path = "@ROCM_PATH@"
+
+# Support substitution of the tools_dir with user parameters. This is
+# used when we can't determine the tool dir at configuration time.
+try:
+    config.llvm_tools_dir = config.llvm_tools_dir % lit_config.params
+    config.llvm_shlib_dir = config.llvm_shlib_dir % lit_config.params
+except KeyError:
+    e = sys.exc_info()[1]
+    key, = e.args
+    lit_config.fatal("unable to find %r parameter, use '--param=%s=VALUE'" % (key,key))
+
+# If rocm_agent_enumerator shows no viable GPUs, skip tests that need one,
+# because the default target will lead to compilation failures.
+config.no_AMD_GPU = False
+config.arch = ""
+config.arch_support_mfma = False
+if config.rocm_path:
+    try:
+        p = subprocess.run([config.rocm_path + "/bin/rocm_agent_enumerator", "-name"],
+                           check=True, stdout=subprocess.PIPE)
+        agents = set(x.decode("utf-8") for x in p.stdout.split())
+        if not agents:
+            # TODO: Remove this workaround for a bug in rocm_agent_enumerator -name
+            # Once https://github.com/RadeonOpenCompute/rocminfo/pull/59 lands
```

Comment:
someone said it's merged in the internal repo. Does it mean we need to remove this workaround?